ui/package.json

```diff
@@ -24,6 +24,7 @@
     "react-transform-catch-errors": "^1.0.2",
     "react-transform-hmr": "^1.0.4",
     "reactable": "^0.14.1",
+    "react-table": "^3.1.0",
```

Comment:
Ah. I thought 3.1.0 was the latest based off the RELEASES tab on their GitHub project

Can try it out l8r